src/main/java/frc/robot/subsystems/Drivetrain.java

```diff
@@ -91,6 +88,15 @@ public void initializeMotorControllers(){
     leftMaster.getEncoder().setPosition(0);
     rightMaster.getEncoder().setPosition(0);
     
+    leftMaster.setOpenLoopRampRate(0.5);
+    rightMaster.setOpenLoopRampRate(0.5);
+
+    leftMaster.setSmartCurrentLimit(30);
```

Comment:
How close is this to allowing the breaker to pop?  Do we need to back off on the right side limits?